### ISSUE TITLE
Syntax highlighting not working?

### ISSUE THREAD

**Issue by 2metres** [cannikin/language-haml#13]
![screen shot 2014-07-14 at 10 44 24 pm](https://cloud.githubusercontent.com/assets/924528/3570735/c257c288-0b54-11e4-9555-61efd7263bcd.png)

Any ideas why this might be happening?

**Comment by anomaly256** [comment]
Not sure if this is the same issue here or something completely new, but I too am not getting any HAML syntax highlighting in atom. In fact, if I ctrl-shift-l to manually select a syntax, HAML does not appear at all.  I've tried uninstalling language-haml then re-installing it, Atom just never seems to pick it up.

I found this in the js console after atom had started:

```
 Failed to load grammar: /home/anomaly/.atom/packages/language-haml/grammars/ruby haml.cson SyntaxError: Syntax error on line 183, column 14: Unexpected token {
```

**Comment by anomaly256** [comment]
Actually I seem to have fixed my issue, which may also fix for others. I've issued a PR which replaces the double quotes with single quotes and removes commas from the offending cson blocks

**Comment by freetonik** [comment]
anomaly256's PR fixes the bug for me (OS X 10.10.1, latest Atom)

**Comment by cannikin** [comment]
Can anyone else confirm this is fixed for them with 0.15.0?

**Comment by dgcliff** [comment]
@cannikin this appears to be fixed for me in 0.15.0, thank you
